server/sonar-web/src/main/js/helpers/urls.js

```diff
@@ -86,5 +86,5 @@ export function getComponentFixedDashboardUrl (componentKey, dashboardKey) {
  * @returns {string}
  */
 export function getComponentDashboardManagementUrl (componentKey) {
-  return '/dashboards?resource=' + encodeURIComponent(componentKey);
+  return window.baseUrl + '/dashboards?resource=' + encodeURIComponent(componentKey);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)